src/main/java/uk/gov/dbt/ndtp/federator/utils/ThreadUtil.java

```diff
@@ -30,9 +30,6 @@
 import java.util.concurrent.ExecutorService;
 import java.util.concurrent.Executors;
 import java.util.concurrent.Future;
-import java.util.concurrent.ScheduledExecutorService;
```

Comment:
Perhaps comment out the imports as well? If we uncomment the code below for whatever reason, there will be import errors